thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/controller.js

```diff
@@ -594,13 +683,14 @@ export default Controller.extend({
           if (wowDetails) {
             curr = wow.currentVal.toFixed(2);
             base = wowDetails.baselineValue.toFixed(2);
-            change = wowDetails.change.toFixed(2);
+            change = (wowDetails.change * 100).toFixed(2);
           }
 
           // Set displayed value properties. Note: ensure no CP watching these props
           set(anomaly, 'shownCurrent', curr);
           set(anomaly, 'shownBaseline', base);
           set(anomaly, 'shownChangeRate', change);
+          set(anomaly, 'changeDirectionLabel', change < 0 ? 'down' : 'up');
```

Comment:
I wonder if the more succinct `setProperties(anomaly, {shownCurrent: curr, ...})` would work here

Reply:
Ah - works great. Thanks!